servers/display_server.h

```diff
@@ -53,6 +53,8 @@ class DisplayServer : public Object {
 	RID _get_rid_from_name(NativeMenu *p_nmenu, const String &p_menu_root) const;
 #endif
 
+	Vector<Object *> additional_outputs;
```

Comment:
Not important but can likely be a `LocalVector`.

Reply:
Maybe also of `ObjectID` instead of `Object *`? We don't reference those pointers so it's not too risky but usually the "safe" pattern for objects is to pass ObjectIDs around.